src/main/thrift/parquet.thrift

```diff
@@ -269,36 +269,43 @@ enum GeometryEncoding {
 
 /**
  * Interpretation for edges of elements of a GEOMETRY logical type. In other
- * words, whether a point between two vertices should be interpolated in
- * its XY dimensions as if it were a Cartesian line connecting the two
- * vertices (planar) or the shortest spherical arc between the longitude
- * and latitude represented by the two vertices (spherical). This value
- * applies to all non-point geometry objects and is independent of the
- * coordinate reference system.
+ * words, whether points between two vertices should be interpolated in their
+ * XY dimensions as if they were on a straight line connecting the two vertices
+ * in Cartesian coordinates (linear) or on the shortest arc on a sphere between
+ * the longitude and latitude represented by the two vertices (pseudo geodesic).
+ * This value applies to all non-point geometry objects.
  *
- * Because most systems currently assume planar edges and do not support
- * spherical edges, planar should be used as the default value.
+ * The LINEAR value is independent of the coordinate reference system.
+ * The PSEUDO_GEODESIC value requires a coordinate reference system associated
```

Comment:
I think the "pseudo geodesic" is also fine, but for reference, S2 calls this a "spherical geodesic":

From http://s2geometry.io/about/overview (in "Design Choices", can't link to the exact section):

> In S2, all edges are “spherical geodesics”, i.e. shortest paths on the sphere

Reply:
I wasn't aware of that, thank for the link. I just checked whether their use of "spherical geodesic" is in the same sense as the proposed "pseudo geodesic". I believe that there is a nuance between S2 usage and this PR. The above-cited S2 overview also said:

> Geocentric and geodetic coordinates are two methods for defining a (latitude, longitude) coordinate system over the Earth’s surface. The question sometimes arises: which system does the S2 library use? The answer is: neither (or both). Points in S2 are modeled as lying on a perfect sphere

Geocentric and geodetic latitudes are the same on a sphere, but differ on an ellipsoid. So saying that S2 uses none or both of them is indeed equivalent to saying that S2 works only with coordinates on a sphere. In other words, if S2 was defining a CRS (I saw none in the overview), it would be required to be associated to a sphere, not to an ellipsoid. In this context, "spherical geodesic" is a true geodesic, with the word "spherical" only emphasizing that they are working on a sphere all the way down to the very definition of what is a latitude. Next to that, the S2 overview also said:

> How points are mapped onto the sphere is outside the scope of S2

In my understanding, the Parquet geometry extension is where this mapping would be defined, and it seems to be implicitly a "do nothing" operation. More advanced applications would, for example, convert geodetic latitudes to some [auxiliary latitudes](https://mathworld.wolfram.com/AuxiliaryLatitude.html) (e.g. authalic latitudes or conformal latitudes) before to give them to S2, but this is another story. The key point is that the decision to approximate an ellipsoid by a sphere (and how to do that) is done in Parquet, not in S2. So the fact that geodesics are not real geodesics was not really relevant to S2, but become relevant to Parquet.

Reply:
The problem with "spherical" is similar to `PLANAR`: it does not distinguish between variants of the "spherical geodesic" interpolation method. A future Parquet version could add `AUTHALIC_GEODESIC` (I don't know if it is okay to put those terms together, we would need to check usage) for meaning _"convert geodetic latitudes to authalic latitudes before to compute geodesics on the authalic sphere"_, or `CONFORMAL_GEODESIC` for meaning _"convert to conformal latitudes etc."_. The problem with `SPHERICAL_GEODESIC` is that it applies to all of them. This is similar to `PLANAR` which applies to linear interpolation, quadratic, cubic, Bézier, _etc._

**Note 1:** users would be interested in the authalic variant if they care about surfaces, or in the conformal variant if they care about shapes. But I don't know if such usage exists, as a better alternative is to use real geodesics. However, such usage exists for the Mercator projection (ESRI's _"Mercator Auxiliary Sphere type 3"_), which make me think that it could exist for geodesics too.

**Note 2:** it is quite possible that the group decides that Parquet will never add such beasts as "authalic geodesic", and instead add real geodesics if demanded. In such case, `SPHERICAL_GEODESIC` is as good as other names. But it would be a constraint on future maintainers. `PSEUDO_GEODESIC` seems more future-proof to me, as it leaves more doors open.

Reply:
I'm not sure to understand. Addition of `Edges` enum values as the right door for more accurate representations in the future was precisely the reason why I suggested to avoid "spherical", as it would clash (i.e., create a confusing situation) with the new enum values…

Reply:
@desruisseaux @paleolimbot Do we achieve any agreement on this?